### PR TITLE
stable/sealed-secrets Upgrade version of sealed-secrets to v0.13.1

### PR DESCRIPTION
**Pull request by krecik**
#### Which issue this PR fixes
Bump version to v0.13.1

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/sealed-secrets/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: sealed-secrets
 description: A Helm chart for Sealed Secrets
-version: 1.11.0
-appVersion: 0.12.4
+version: 1.12.0
+appVersion: 0.13.1
```
Comment (olib963) [inline stable/sealed-secrets/Chart.yaml:4]: I think you need to at least update the tag in `values.yaml`. I will take a longer look at this later.

Comment (krecik) [reply stable/sealed-secrets/Chart.yaml:4]: Updated. Missed this file in first attempt :)

**Review by olib963 — APPROVED** [review]
/lgtm

**Pull request merged**